CHANGELOG.md

```diff
@@ -4,7 +4,12 @@
 
 * Added support for `original_transaction` to `Recurly_Transaction` [#238](https://github.com/recurly/recurly-client-php/pull/238)
 * Added `Recurly_AccountBalance` [#239](https://github.com/recurly/recurly-client-php/pull/239)
-* Print warnings when using a deprecated version of the API.
+* Print warnings when using a deprecated version of the API. [#250](https://github.com/recurly/recurly-client-php/pull/250):
```

Comment:
I realized this one didn't have a PR links so I just stuck it in here while rebasing.